src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.tsx

```diff
@@ -62,10 +85,25 @@ const CreateOrEditNote = ({
       sx={styles.container}
     >
       <Box sx={styles.header}>
-        <Avatar />
-        <Typography variant={TypographyVariantEnum.Subtitle2}>
-          {userNameMocked}
-        </Typography>
+        {loading ? (
+          <Loader size={20} />
+        ) : (
+          <>
+            <Avatar
+              src={
+                photo
+                  ? `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`
```

Comment:
You can replace it with following code:

```jsx
photo ?? `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`
```

We discussed it on the meeting

Reply:
Yes, we discussed it, but I decided to use a different way of specifying the condition because of the errors that occurred